docs/repo-guides/release-nnf-sw/readme.md

```diff
@@ -0,0 +1,161 @@
+    * [NearNodeFlash/nnf-sos](https://github.com/NearNodeFlash/nnf-sos)
+    * [HewlettPackard/dws](https://github.com/HewlettPackard/dws)
+    * [HewlettPackard/lustre-csi-driver](https://github.com/HewlettPackard/lustre-csi-driver)
+    * [NearNodeFlash/lustre-fs-operator](https://github.com/NearNodeFlash/lustre-fs-operator)
+- [NearNodeFlash/NearNodeFlash.github.io](https://github.com/NearNodeFlash/NearNodeFlash.github.io)
+
+[nnf-ec](https://github.com/NearNodeFlash/nnf-ec) is vendored in as part of `nnf-sos` and does not
+need to be released separately.
+
+## Primer
+
+This document is based on the process set forth by the [DataWorkflowServices Release
+Process](https://dataworkflowservices.github.io/v0.0.1/repo-guides/create-a-release/readme/#github-release-process).
+Please read that as a background for this document before going any further.
+
+## Requirements
+
+To create tags and releases, you will need maintainer or admin rights on the repos.
+
+## Release Each Component In `nnf-deploy`
+
+As mentioned above, you'll first need to create releases for each component contained in
+`nnf-deploy`. This section describes that process.
+
+Each release branch needs to be updated with what is on master. To do that, we'll need the latest
+copy of master, and it will ultimately be merged to the `releases/v0` branch via a Pull Request.
+Once merged, a tag is created and then a release.
+
+<details>
+<summary>Component Version Numbers</summary>
+Each component has its own version number that needs to be incremented. Make sure you change the
+version numbers in the commands below to match the new version for the component. The `v0.0.3` is
+just an example.
+</details>
+
+1. Ensure your branches are up to date:
+
+    ```shell
+    git checkout master
+    git pull
+    git checkout releases/v0
+    git pull
+    ```
+
+2. Create a branch to merge into the release branch:
+
+    ```shell
+    git checkout -b release-v0.0.3
+    ```
+
+3. Merge in the updates from the `master` branch. There **should not** be any conflicts, but it's
+   not unheard of. Tread carefully if there are conflicts.
+
+    ```shell
+    git merge master
+    ```
+
+4. Verify that there are no differences between your branch and the master branch:
+
+    ```shell
+    git diff master
+    ```
+
+    If there are any differences, they must be trivial. Some READMEs may have extra lines at the
+    end.
+
+5. For `lustre-csi-driver` and `lustre-fs-operator`, there are additional files that need to track
+   the version number as well.
+
+    a. For `lustre-fs-operator`, update `config/manager/kustomization.yaml` with the correct
+    version.
+
+    b. For `lustre-csi-driver`, update `deploy/kubernetes/base/kustomization.yaml` and
+    `charts/lustre-csi-driver/values.yaml` with the correct version.
+
+6. Create a Pull Request from your branch and **target the release branch**. When merging the Pull
+   Request, **you must use a Merge Commit. Do not Rebase or Squash.**
```

Comment:
We should explain the reason for this:

```suggestion

!!! note

    Do not Rebase or Squash!  Those actions will remove the records that Git
    uses to determine which commits have been merged, and then when the next
    release is created Git will treat everything like a conflict.

```

Reply:
Note that VSCode will render that `!!! note` line explicitly.  That's okay.  Because 'mike' will render it as a "note" with a little pencil icon next to it.  See it how a `!!! note` looks here: https://nearnodeflash.github.io/v0.0.2/guides/user-containers/readme/